src/main/java/JavaTutorial/LinkedList1.java

```diff
@@ -0,0 +1,24 @@
+package JavaTutorial;
+
+import java.util.LinkedList;
+
+public class LinkedList1 {
```

Comment:
Same as comment given above for class name

Reply:
Resolved the changes.